src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,196 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_LOGIN_FOR_TESTS = "validLogin";
+    private static final String DEFAULT_PASSWORD_FOR_TESTS = "validPassword";
+    private static final int DEFAULT_AGE_FOR_TESTS = 18;
+    private static RegistrationServiceImpl registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin(DEFAULT_LOGIN_FOR_TESTS);
+        user.setPassword(DEFAULT_PASSWORD_FOR_TESTS);
+        user.setAge(DEFAULT_AGE_FOR_TESTS);
+    }
+
+    @Test
+    void userExistsAndLoginAndPasswordAreValid_Ok() {
+        User registeredUser = registrationService.register(user);
+        assertNotNull(registeredUser);
+        assertEquals(user, registeredUser);
+        assertTrue(Storage.people.contains(registeredUser));
+    }
+
+    @Test
+    void userDoesNotExist_NotOk() {
+        User actual = new User();
+        assertFalse(Storage.people.contains(actual));
+    }
```

Comment:
this test is wrong and redundant. because we don't call any service method here.
please check all similar cases and remove redundant test methods that don't have call service method inside.